src/app/public/modules/dropdown/dropdown.component.spec.ts

```diff
@@ -146,6 +173,20 @@ describe('Dropdown component', function () {
     expect(button).toHaveCssClass('sky-btn-default');
   }));
 
+  it('should change theme', fakeAsync(() => {
```

Comment:
What's this actually testing? We already know the theme service works... is this necessary?

Reply:
Updated to actually test something.